enterprise/kubernetes.md

```diff
@@ -0,0 +1,108 @@
+# Installing Private Packagist Enterprise on Kubernetes
+
+Private Packagist Enterprise can run on a Kubernetes cluster. To do that, Private Packagist Enterprise uses Replicated.
+
+Replicated is the application which will provide you with a management interface to your Private Packagist Enterprise installation and allow you to monitor the task and perform maintenance operations such as backups or updates.
+
+## General requirements
+
+1. License Key File (file extension .rli) Don't have one yet? [Sign up for a free trial license!](https://packagist.com/enterprise)
+2. One (sub-)domain to operate the web interface, e.g. packagist.myintranet.com
+3. One (sub-)domain to operate the composer repository, e.g. repo.packagist.myintranet.com or packagist-repo.myintranet.com
+4. An SSL certificate valid for both chosen domains
+5. An SMTP server or a GMail account for Private Packagist Enterprise to send email
+
+## Options
+
+There are two options for running Private Packagist Enterprise on a Kubernetes cluster:
+
+* **Start from scratch.** Installing Replicated Kubernetes on a Linux server and running Private Packagist Enterprise there
+* **Use an existing Kubernetes cluster.** Installing Replicated on your existing Kubernetes cluster and running Private Packagist Enterprise there
+
+## Installing Private Packagist Enterprise from scratch
+
+### Requirements
+
+* A Linux Server
+  * OS
+    * Ubuntu 16.04.3 - 16.04.5
+    * Red Hat Enterprise Linux 7.4 - 7.5
+    * CentOS 7.4 - 7.5
+  * At least 4GB memory
+  * At least 2 CPU cores
+  * At least 40GB disk space
+  * Ports 443 and 8800 must be accessible
+  * Must be reachable at the chosen domain names from your local machine
+
+### Installation
+
+The easiest way to install Replicated Kubernetes is the easy install script. You can run it using the following two commands:
+
+```
+curl -sSL -o install.sh https://get.replicated.com/kubernetes-init
```

Comment:
Think the URL here should be https://get.replicated.com/kubernetes-init/privatepackagist/stable similar to what we have for the Native Scheduler, this isn't really well documented by Replicated, but it ensures the version of Replicated gets installed which we are using on our latest stable release.

Reply:
If I try https://get.replicated.com/kubernetes-init/privatepackagist/stable I get this at the bottom:

```bash
printf "${RED}
            404: Not Found
            There was an error processing the request to generate the installation script

                Request path: $PATH

            Please visit https://help.replicated.com/ for installation guides and documentation${NC}\n\n\
"

exit 1
```

so I don't think that works?

Reply:
Yeah might be for the native scheduler only, so let's just leave it like this, I may check with them at some later point.

Reply:
Actually found the correct instructions on their admin UI:

```
curl -sSL -o install.sh https://get.replicated.com/privatepackagistkubernetes/stable/kubernetes-init
sudo bash install.sh
```

URL just looked a bit different than expected ;-)